server/raft.go

```diff
@@ -4229,7 +4226,7 @@ func (n *raft) updateLeadChange(isLeader bool) {
 }
 
 // Lock should be held.
-func (n *raft) switchState(state RaftState) {
+func (n *raft) switchState(state RaftState) (leadChange bool) {
```

Comment:
Bit of a nit, but wondering if this needs to be a named return.

Reply:
Doesn't need to be, have removed it.